src/test/java/com/puppycrawl/tools/checkstyle/api/FileContentsTest.java

```diff
@@ -280,7 +280,7 @@ public void testHasIntersectionEarlyOut() throws Exception {
                 "clangComments");
         final TextBlock textBlock = new Comment(new String[] {""}, 1, 1, 1);
         clangComments.put(1, Collections.singletonList(textBlock));
-        clangComments.put(2, null);
+        clangComments.put(2, Collections.emptyList());
```

Comment:
resolves `FileContentsTest.testHasIntersectionEarlyOut:285 » NullPointer`

Reply:
This now causes a pitest failure.
https://app.circleci.com/pipelines/github/checkstyle/checkstyle/4214/workflows/227adf91-df2d-4285-8fda-f24cbd7c8afa/jobs/69854
````
+FileContents.java.html:<td class='covered'><pre><span  class='survived'>            if (hasIntersection) {</span></pre></td></tr>
````

Reply:
Fixed.